projects/v3/src/app/pages/chat/chat-room/chat-room.component.ts

```diff
@@ -775,4 +774,9 @@ export class ChatRoomComponent implements OnInit {
       await modal.present();
     }
   }
+
+  onContentChangedQull(data) {
+    console.log(data);
+  }
```

Comment:
This one still necessary? @sasangachathumal

Reply:
no. will remove that.